charts/trento-server/charts/trento-wanda/Chart.yaml

```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.3.0
+version: 1.3.1
```

Comment:
Typo?
We have not updated wanda

Reply:
Well, true... :smile: I guess it's not needed